workflow.go

```diff
@@ -215,7 +215,7 @@ func runOnce(
 		// Exit cleanly if error returned is cancellation of context
 		return err
 	} else if err != nil {
-		logger.Error(ctx, fmt.Errorf("run error [role=%v]: %v", role, err))
+		logger.Error(ctx, fmt.Errorf("run error [role=%v], [process=%v]: %v", role, processName, err))
```

Comment:
do you mind updating these to use %s over %v while you have this open please? Maybe leave error as %v even though errors should always be described as a string.

Reply:
done ✅